src/taxpasta/infrastructure/application/kraken2_profile_reader.py

```diff
@@ -48,6 +48,7 @@ def read(cls, profile: ProfileSource) -> DataFrame[Kraken2Profile]:
             sep="\t",
             header=None,
             index_col=False,
+            dtype={"taxonomy_id": str},
```

Comment:
Should this line be added to the other classifiers as well?

Reply:
Yes, good point. I will make another PR and update the other profiles.

Reply:
Sounds good!